### PR TITLE
Added transitions

### PR DESCRIPTION
**Pull request by ashkrao**

### CODE REVIEW

File: CSS/style.css [hunk 1]
```diff
@@ -0,0 +1,30 @@
+.prodImage {
+  width: 30%;
+  height: 400px;
+  display: inline-block;
+  vertical-align: middle;
+  background-color: #ddd;
```
Comment (dbiehle) [inline CSS/style.css:6]: What does background-color on an image look like? Or does this class 'prodImage' reference a container of the image that could accept a background-color?

File: js/app.js [hunk 2]
```diff
@@ -87,12 +87,15 @@ function removeExistingImages() {
 
 function renderBusMallImages (item) {
   var busMallImagesParent = document.getElementById('busMallImagesParent');
+  var imgSpan = document.createElement('span');
```
Comment (dbiehle) [inline js/app.js:90]: I'm just making sure I understand this section correctly, so let me know if/where I'm wrong. You added a <span> element to the busMallImagesParent and added a new class to it called prodImage. Then in CSS, this is the element that will get a transition when hovered over. The span element is created as a container for the image. Is that right?

**Review by dbiehle — COMMENTED** [review]
Looks great! I just added a couple of questions so I can better understand the code/changes I looked at.

File: js/app.js (hunk 2 above)
Comment (ashkrao) [reply js/app.js:90]: Yes, the span is a container for the image. I added the span to draw a uniform box around each image. In CSS I have set transitions on hover for both the image and the span.

File: CSS/style.css (hunk 1 above)
Comment (ashkrao) [reply CSS/style.css:6]: I have set 'prodImage' class for spans. They are containers for images.

**Pull request merged**